pkg/server/server.go

```diff
@@ -178,6 +180,8 @@ func NewServer(c CompletedConfig) (*Server, error) {
 		}
 	}
 
+	s.DynRESTMapper = dynamicrestmapper.NewDynamicRESTMapper(nil)
```

Comment:
So this would be global mapper every controller can pull In? Can we add a comment here tell how this can be used?